src/main/java/io/passport/server/service/ModelService.java

```diff
@@ -32,6 +32,7 @@ public List<Model> getAllModels(){
         return modelRepository.findAll();
     }
 
+
```

Comment:
Newline left from deleted content. You can remove them along with the other marked newlines.